backend/src/router/userRouter.ts

```diff
@@ -0,0 +1,9 @@
+import * as express from 'express';
+import userController from '../controllers/userController';
+import { AuthMiddleware } from '../middlewares/authMiddleware';
+
+const router = express.Router();
+
+router.get('/users', AuthMiddleware, userController.getUsers);
```

Comment:
What is the reason to return all users from API?

Reply:
I only used it to test authMiddleware, but I can change to return user by id or email

Reply:
Don't push code, that was written for testing purposes and won't be used for integration